src/cdk/drag-drop/drag.ts

```diff
@@ -751,3 +751,11 @@ interface Point {
   x: number;
   y: number;
 }
+
+/** Creates a deep clone of an element. */
+function deepCloneNode(node: HTMLElement): HTMLElement {
+  const clone = node.cloneNode(true) as HTMLElement;
+  // Remove the `id` to avoid having multiple elements with the same id on the page.
```

Comment:
Could this cause issues if the element is styled based on the `id`?

Should we at least document this in the `md` file for dragdrop?

Reply:
I'll add a note about it, but we've been doing this in the `MatIconRegistry` for a long time now and it hasn't come up.